test/integration/test_fake_backends.py

```diff
@@ -164,3 +164,8 @@ def test_non_cx_tests(self):
         self.assertIsInstance(backend.target.operation_from_name("cz"), CZGate)
         backend = FakeSherbrooke()
         self.assertIsInstance(backend.target.operation_from_name("ecr"), ECRGate)
+
+    def test_retrieving_single_backend(self):
+        backend_name = "fake_jakarta"
+        backend = FAKE_PROVIDER_FOR_BACKEND_V2.backend(backend_name)
+        self.assertEqual(backend.name, backend_name)
```

Comment:
I think that this is a unit test rather than an integration test, I would move it to `test/unit/fake_provider/test_fake_backends.py`